env/parameters.schema.json

```diff
@@ -40,11 +39,6 @@
           "title": "Pipeline bot Git username",
           "description": "The Git user that will perform git operations inside a pipeline. It should be a user within the Git organisation/owner where environment repositories will live. This is normally a bot."
         },
-        "email": {
```

Comment:
Looks like some changes are being reverted, possibly just needs rebasing.

Reply:
I am not sure, but it seems to be removed also form here https://github.com/jenkins-x/environment-tekton-weasel-dev/blob/676ef0e92724ed89ae6193aa5a7ac20a2b01b1f8/env/parameters.schema.json#L31

Was this accidentally done?

Reply:
I think the weasel environment was created before this PR was merged https://github.com/jenkins-x/jenkins-x-boot-config/pull/44/files

Reply:
I reverted the changes to include the `email` field.